docs/units/index.rst

```diff
@@ -238,6 +238,7 @@ Reference/API
 .. automodapi:: astropy.units.cgs
 
 .. automodapi:: astropy.units.astrophys
+   :skip: littleh
```

Comment:
These slightly argue for not having `littleh` inside `astropyphys` at all any more (or at least not in `__dir__`)

Reply:
Removing ``__dir__``  solved the problem!

Reply:
So, this line can now be removed?

Reply:
I thought I had! I used a ``fixup!`` commit to prune the whole commit history.

Reply:
✅